internal/page/data_test.go

```diff
@@ -147,6 +147,18 @@ func TestReplacementAttorneysStepIn(t *testing.T) {
 	})
 }
 
+func TestGenerateHash(t *testing.T) {
+	lpa := &Lpa{}
+	hash, err := lpa.GenerateHash()
+	assert.Nil(t, err)
+	assert.Equal(t, uint64(6045214389284053903), hash)
+
+	lpa.ID = "1"
+	hash, err = lpa.GenerateHash()
+	assert.Nil(t, err)
+	assert.Equal(t, uint64(10213271685641464943), hash)
```

Comment:
these numbers will change when we add/remove fields from Lpa, but I think that is ok because at least we'll know it is working